### ISSUE TITLE
Can this be ported to React js?

### ISSUE THREAD

**Issue by romashka85** [AbhilashJN/react-native-word-cloud#1]
Hello, I just wondered if this generator can be ported to React js and if yes? What should I consider if I do?

**Comment by AbhilashJN** [comment]
Yes absolutely. Some minor tweaks may be required. You are looking to draw this in a canvas right?
